microsoft-edge/extensions-chromium/publish/add-ons-store-curation.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Edge Web Store Curation and Reviews
+description: How submissions to the Microsoft Edge Web Store are curated and reviewed by the store team.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 10/13/2023
+---
+# Edge Web Store Curation and Reviews
+
+The Microsoft Edge Add-Ons Store has curated the Edge store experience so that users can find all items like themes, extensions, and apps to customize their browser at one place.
+
+Before installing an item from the Edge Add-Ons Store, users can learn more about the functionality, data permissions, and publisher by viewing the description and listing.
+
+
+<!-- ====================================================================== -->
+## Organizing Edge Add-Ons Store items
+
+The store organizes the items to help users discover the ones that are relevant to their interests.
+
+Users can also discover the items they are interested in on the Edge Add-Ons Store in several ways such as the home page, search bar, multiple curated collections, and categories.
+
+The following factors are used to organize items:
+* **Quality:** Items featured on the Edge Add-Ons Store are selected based on their eligibility criteria.
+* **Relevance:** Ranking takes into consideration factors such as the item name, description relevancy, popularity, and user experience.
```

Comment:
@Bhuvana-Priya - .docx has "relevancy".  Either word seems ok; I might research more about the two words.
https://ahdictionary.com/word/search.html?q=relevancy